controllers/topics/inviteMembers.js

```diff
@@ -111,8 +108,6 @@ module.exports = async (req, res) => {
               .then(() => {});
           });
         }
-
-        return user_id;
       })
     );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Removing the return statement at the end of the Promise.all map function may cause unexpected behavior. If you need to use the result of this operation later in your code, it would be better to keep the return statement.
```diff
- // No return statement
+ return user_id;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->